base/string_utils.cpp

```diff
@@ -326,4 +326,29 @@ bool AlmostEqual(string const & str1, string const & str2, size_t mismatchedCoun
   return false;
 }
 
+void Split(string const & s, char delimiter, vector<string> & target)
```

Comment:
1. SimpleTokenizer may be re-used to work on std::string
2. It omits delimiters.

Please, take a look at tests for TokenizeIterator. I hope it does what do you need.